src/plugins/workspace/public/components/workspace_list/index.tsx

```diff
@@ -38,23 +44,83 @@ export interface WorkspaceListProps {
   registeredUseCases$: BehaviorSubject<WorkspaceUseCase[]>;
 }
 
+interface WorkspaceAttributeWithUseCaseID extends WorkspaceAttribute {
+  useCase: string[];
+}
+
 export const WorkspaceList = ({ registeredUseCases$ }: WorkspaceListProps) => {
   const {
     services: { workspaces, application, http },
   } = useOpenSearchDashboards();
   const registeredUseCases = useObservable(registeredUseCases$);
   const isDashboardAdmin = application?.capabilities?.dashboards?.isDashboardAdmin;
-
   const initialSortField = 'name';
   const initialSortDirection = 'asc';
   const workspaceList = useObservable(workspaces?.workspaceList$ ?? of([]), []);
-  const [queryInput, setQueryInput] = useState<string>('');
+
+  const newWorkspaceList: WorkspaceAttributeWithUseCaseID[] = useMemo(() => {
+    return workspaceList.map(
+      (workspace): WorkspaceAttributeWithUseCaseID => ({
+        ...workspace,
+        useCase: [...(workspace.features || [])],
```

Comment:
We can make it a string instead of array, then you don't need to call `getFirstUseCaseOfFeatureConfigs` many times
```suggestion
        useCase: getFirstUseCaseOfFeatureConfigs(workspace.features || []) ?? ''
```

Reply:
Seems there is bug when it's an array, see my screenshot
![image](https://github.com/user-attachments/assets/4ebaac98-b891-40a1-8c88-a76b08c11ef9)

Reply:
Thank you for your comments. I have updated the array to a string and addressed the other issues mentioned. Could you please review the changes and let me know if the bug still persists? Thank you!